mockgen/mockgen.go

```diff
@@ -171,7 +159,28 @@ func main() {
 	if err := g.Generate(pkg, outputPackageName, outputPackagePath); err != nil {
 		log.Fatalf("Failed generating mock: %v", err)
 	}
-	if _, err := dst.Write(g.Output()); err != nil {
+	output := g.Output()
+	dst := os.Stdout
+	if len(*destination) > 0 {
+		if err := os.MkdirAll(filepath.Dir(*destination), os.ModePerm); err != nil {
+			log.Fatalf("Unable to create directory: %v", err)
+		}
+		existing, err := os.ReadFile(*destination)
```

Comment:
This all seems reasonable to me, would you mind using an older API that is 1.15 compatible. After the next release I plan on dropping 1.15 support. But for now this would be nice to land in the next release.

Reply:
Done